src/raw/reply.rs

```diff
@@ -19,26 +19,32 @@ use crate::{FileType, Result};
 ///
 /// Nearly the same as a `libc::timespec`, except for the width of the nsec
 /// field.
+///
+/// ## Note:
+///
+/// Limited by Linux FUSE protocol, any timestamp earlier than UNIX_EPOCH
```

Comment:
This statement is not true.  I just double-checked it on FreeBSD, and I don't think it's true of Linux either.  Negative timestamps work fine.

Reply:
but in the Linux `fuse.h`, the type of `atime`, `mtime`, `ctime` is `uint64_t`, how do we pass a negative timestamp to the kernel?

Reply:
Simply by casting.
```rust
let true_ts: i64 = -100_000;
let atime: u64 = true_ts as u64;
```